plugins/available/java.plugin.bash

```diff
@@ -1,11 +1,11 @@
-cite about-plugin
+# shellcheck shell=bash
 about-plugin 'Java and JAR helper functions'
 
 function jar_manifest {
-  about "extracts the specified JAR file's MANIFEST file and prints it to stdout"
-  group 'java'
-  param '1: JAR file to extract the MANIFEST from'
-  example 'jar_manifest lib/foo.jar'
+	about "extracts the specified JAR file's MANIFEST file and prints it to stdout"
+	group 'java'
+	param '1: JAR file to extract the MANIFEST from'
+	example 'jar_manifest lib/foo.jar'
 
-  unzip -c $1 META-INF/MANIFEST.MF
+	unzip -c "${1?${FUNCNAME[0]}: JAR file must be specified}" META-INF/MANIFEST.MF
```

Comment:
My point was, when doing a `?` check, unless an empty (`""`) value is meaningful, use `:?` to catch the pesent-but-empty value case. Here, an empty value does not appear to be meaningful.

i.e. if the user typed (or essentially resolved to) `jar_manifest ""`, we would want to fail with an error message, yes?

Reply:
Ah! I misunderstood!! I alsö misread! And alsö mistyped! Sorry for the static...

Reply:
Fixed!